### DISCUSSION TITLE
How to migrate from CRA to Turbopack?

### DISCUSSION THREAD

**Discussion by amarg26** [vercel/turborepo#2458] [Help] [answered]
How to migrate from CRA to Turbopack? Need steps.

**Comment by jridgewell** [comment, marked as answer]
We're currently in alpha, and and the custom build code that CRA uses is not compatible with our dev server.

In the future it will be possible to migrate from CRA to Turbopack. But it'll also possible for CRA to use Turbopack internally instead, so you wouldn't need to migrate at all.

**Reply by amarg26** [reply]
@jridgewell It means we can use it with react.js just like vite.js once it becomes stable.
